src/bin/node.rs

```diff
@@ -299,7 +305,8 @@ fn run(
     Ok(())
 }
 
-fn main() {
+#[tokio::main]
+async fn main() {
```

Comment:
Can we spawn the tokio runtime outside of the main thread so we can get clear separation between whatever runtime the kernel uses internally and to power its interfaces, and the tokio runtime required by capnp? The way I am reading this now, we're going full tokio on the entire runtime, but do we really want to commit to that?